pull_request_template.md

```diff
@@ -1,15 +1,9 @@
-## What did you do? 
-- [describe the proposed changes]
-- 
-
-
-## Why did you make this change?
-(Cite Issue number OR provide rationalization of changes if no issue exists)
+(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the changes. The "why" may simply be citing a relevant Galaxy issue.)
 (If fixing a bug, please add any relevant error or traceback)
+(For UI components, it is recommended to include screenshots or screencasts)
 
-
-## How to test the changes? 
-(select the most appropriate option; if the latter, provide steps for testing below)
+## How to test the changes?
```

Comment:
Could there be cases where more than 1 case applies?

Reply:
We could say "at least one", but I never needed more than one.

Reply:
"select all options that apply"? I also had more than one (refactoring, tests exist that cover part of the modified code + I added more tests)

Reply:
Aren't we going back to the version that was confusing to some people?

Reply:
I think the confusion was that the 3rd option could be used for release testing. Writing that in a way that anyone can take this out of context and test it would be a burden for the PR author writing it, and probably not a good way of spending time while testing the release. It should be sufficient for reviewers to understand why the change was made, and IMO can also just point to an issue if it exists.

Saying `select all options that apply` is less ambiguous than picking one, since picking one is tricky for many PRs.

Reply:
I've used multiple options on occasion also. I like language allowing multiple options.